.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -44,6 +44,22 @@ Paste the output of "ttn-lw-cli version" or "ttn-lw-stack version" if applicable
 
 ...
 
+#### How do you propose to test this?
+<!--
+Please think about how this is verified as implemented?
+For example, you can add what commands need to be run via the CLI/Console to verify that this works.
+-->
+
+...
+
+##### What else needs to be checked as regression?
+<!--
+Please think about things that this change could affect and how that can be tested?
+You can also @mention experts if you need help with defining this.
+-->
+
+...
+
```

Comment:
I don't think feature requesters can answer this at the time they submit the issue.

Reply:
Agreed. In fact, I would move this section to the PR template, as the PR author may have a sense for potential regression elsewhere